Steepfile

```diff
@@ -4,7 +4,6 @@ target :lib do
   signature "sig"
 
   check "lib"
-  library "rubygems"
```

Comment:
Fix the warning message from steep:
```
W, [2023-09-11T13:06:25.229996 #30196]  WARN -- rbs: `rubygems` has been moved to core library, so it is always loaded. Remove explicit loading `rubygems`
```

Reply:
Sounds good. I will do it on a separate PR, as RBS 3.0 has some breaking changes. https://github.com/ruby/rbs/wiki/Release-Note-3.0

Also, it reduces the scope of this PR 😄